pkgs/development/interpreters/python/hooks/pytest-check-hook.sh

```diff
@@ -33,6 +33,10 @@ function pytestCheckPhase() {
 
     # Compose arguments
     args=" -m pytest"
+    if [ -n "$dontIgnoreWarnings-}" ]; then
```

Comment:
This reads as "if not empty don't ignore warnings", which is a lot of negatives. Would `if [ -z "$showWarnings-}" ]` (which can be read as "if empty show warnings") be possible instead?